src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,66 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__header {
+		display: flex;
+	}
+
+	.tribe-events-calendar-month__header-column {
+		@mixin desktop-body-3;
+		text-transform: uppercase;
+		flex: 1 1 0;
+		padding-bottom: var(--spacer-2);
+	}
+
+	.tribe-events-calendar-month__week {
+		display: flex;
+
+		&:first-of-type {
+			@media (--viewport-medium) {
+				border-top: 1px solid var(--color-border-default);
+			}
+		}
+	}
+
+	.tribe-events-calendar-month__day {
+		flex: 1 1 0;
+		font-family: var(--font-family-base);
+		@media (--viewport-medium) {
+			border-width: 0 1px 1px 0;
+			border-style: solid;
+			border-color: var(--color-border-default);
+			min-height: 168px;
+		}
+
+		&:hover {
+			@media (--viewport-medium) {
+				border-bottom: 1px solid var(--color-border-active);
+				box-shadow: inset 0 -1px 0 0 var(--color-border-active);
+			}
+		}
+
+		&:first-of-type {
+			@media (--viewport-medium) {
+				border-left: 1px solid var(--color-border-default);
```

Comment:
Let's put this style on the row instead. If we do this, the first cell will be 1px smaller than the others (due to the border).